### DISCUSSION TITLE
catalogue

### DISCUSSION THREAD

**Discussion by KcVamsi** [learndevopsonline/learndevopsonline#604] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/129276861/232996364-1d00263b-4175-4fd7-962c-852fdc340539.png)
![image](https://user-images.githubusercontent.com/129276861/232996442-1590e48d-f658-4701-8351-bc9d5e09ea89.png)
Hello,

Even after copying the file from the to location my mongodb is getting error.
But copying was successful.

**Comment by naveen2513** [comment, marked as answer]
24. cp /home/centos/roboshopusing-shell/mogodb.repo  /etc/yum.repos.d/mongo.repo
